chore(profile): add profile for Muhammad Ramadhani

dante4rt opened the pull request:
### Linked Issues
Closes holdex/hr-member-muhammad-ramadhani#1 

This pull request updates the `profile-submission.json` file to keep the trial participant list current. The most important changes are:

**Participant Addition:**
Added a new entry for `dante4rt` (Muhammad Ramadhani) to the participants list.

<!-- This is an auto-generated comment: release notes by coderabbit.ai -->
## Summary by CodeRabbit

- Chores
  - Added a new team profile for Muhammad Ramadhani to the public team list.
  - Ensured the new profile appears in the correct position and includes the trial issue link.
  - No functional changes to the application; the visible team directory now reflects the updated membership.
<!-- end of auto-generated comment: release notes by coderabbit.ai -->

tiffmyaaw reviewed (changes requested):
1. Past commits are still missing `verified` handle. Kindly follow [this instruction](https://gist.github.com/sbolel/589af0ec390d4c1460137958fc0fea12) to verify past commits. [See also the Guidelines](https://github.com/holdex/developers/blob/main/.github/CONTRIBUTING.md#commit-signature-verification)
1. Edit the PR description to [link your issue manually](https://docs.github.com/en/issues/tracking-your-work-with-issues/using-issues/linking-a-pull-request-to-an-issue).
1. Update your time spent on the PR Time Tracker bot.

Feel free to look at the successfully closed PRs in this repo for refs. Installing our GitHub Wizard may help:
- https://github.com/holdex/github-extension

dante4rt commented:
> 1. Past commits are still missing `verified` handle. Kindly follow [this instruction](https://gist.github.com/sbolel/589af0ec390d4c1460137958fc0fea12) to verify past commits. [See also the Guidelines](https://github.com/holdex/developers/blob/main/.github/CONTRIBUTING.md#commit-signature-verification)
> 2. Edit the PR description to [link your issue manually](https://docs.github.com/en/issues/tracking-your-work-with-issues/using-issues/linking-a-pull-request-to-an-issue).
> 3. Update your time spent on the PR Time Tracker bot.
> 
> Feel free to look at the successfully closed PRs in this repo for refs. Installing our [GitHub Wizard](https://github.com/holdex/github-extension) may help.

Thanks for the feedback!

I've already verified the past commits—feel free to check the screenshot I attached. For number 2 and 3, I've fixed the PR description with the issue link and updated the time spent on the Time Tracker bot.

Kindly take another look when you can, ya? Thanks!

<img width="1088" height="423" alt="Screenshot at Sep 19 09-03-06" src="https://github.com/user-attachments/assets/5f939ba2-7ad7-4552-a5bd-83665a2b1be3" />
